librad/src/net/protocol.rs

```diff
@@ -302,8 +302,8 @@ where
         mut incoming: BoxStream<'_, quic::Result<quic::Stream>>,
         hello: impl Into<Option<gossip::Rpc<SocketAddr, A>>>,
     ) {
-        tracing::info!("New outgoing connection");
         let remote_id = conn.remote_peer_id().clone();
+        tracing::info!(remote_id = %remote_id, "New outgoing connection");
```

Comment:
It wasn't -- I think it's because the span is at trace level, but this at info. So, if you run with `RUST_LOG=info`, the span context will not be logged.

Reply:
Ah, I thought trace _implied_ info

Reply:
Oh no, wrong way around :sweat_smile:

Reply:
Ya it's a very empirical business to get this right... :confused:

Maybe it would be cool to have a formatter which indents the span context, like

```
stuff{fields}
    other_stuff{more}
        even_more_stuff{lots}
```

Reply:
That would be slick indeed :eyes: